src/modules/user/controller.js

```diff
@@ -0,0 +1,270 @@
+import {
+	wrapBody as wrap,
+} from '../util/helper';
+import * as service from './service';
+import errors from '../core/errors';
+import {
+	trim,
+	startsWith, 
+} from 'lodash';
+import { User } from '../models';
+
+export async function create(ctx, next) {
+	let data = null;
+	let error = null;
+    let transaction;
+	try {
+        transaction = await service.transaction();
```

Comment:
Transactions are used in `create`, `update`, `remove`, is it necessary? What is the underlying SQL statements in Sequelize `User.create`?

Reply:
It’s necessary because these actions would modify data
User is import from arc/modules/models/user, and in this file return a MySQL model insatance using sequelize.define() function.So u can use MySQL model inner function or any other defined static function in this file from any where when imported it